src/components/PhotosGallery/PhotosGallery.jsx

```diff
@@ -0,0 +1,124 @@
+import Image from "next/image"
+import React, { useEffect, useState } from "react"
+
+function PhotosGallery(props) {
+  const [file, setFile] = useState(null)
+
+  const onChangeImageHandler = (e) => {
+    if (e.target.files[0]) {
+      const reader = new FileReader()
+      reader.onload = () => {
+        setFile(reader.result)
+      }
+      reader.readAsDataURL(e.target.files[0])
+      e.target.value = ""
+    }
+  }
+
+  useEffect(() => {}, [file])
+
+  function dropLargeImage() {
```

Comment:
Why you made it twice while you can make it one time and render everything in it ?